modules/ascat/main.nf

```diff
@@ -85,16 +87,16 @@ process ASCAT {
 
     #Load the data
     ascat.bc = ascat.loadData(
-        Tumor_LogR_file = "Tumour_tumourLogR.txt",
-        Tumor_BAF_file = "Tumour_normalBAF.txt",
-        Germline_LogR_file = "Tumour_normalLogR.txt",
-        Germline_BAF_file = "Tumour_normalBAF.txt",
+        Tumor_LogR_file = paste0("$prefix", "_tumour_tumourLogR.txt"),
+        Tumor_BAF_file = paste0("$prefix", "_tumour_tumourBAF.txt"),
+        Germline_LogR_file = paste0("$prefix", "_tumour_normalLogR.txt"),
+        Germline_BAF_file = paste0("$prefix", "_tumour_normalBAF.txt"),
```

Comment:
```suggestion
        Germline_LogR_file = paste0("$prefix", ".normal.LogR.txt"),
        Germline_BAF_file = paste0("$prefix", ".normal.BAF.txt"),
```

would this also work? Or do you then run again into issues of files having the same name?

Reply:
Unfortunately, that would not work because of the single testdata prefix (`meta.id`). 
The `prepareHTS` function takes a `tumourname` and`normalname` as prefix to name the LogR and BAF files and renaming them would in theory work (the `normalLogR`/`normalBAF` and `tumourLogR`/`tumorBAF` is internally attached and cannot be changed) .
The `alleFrequency` files are also named according to the `tumourname` and`normalname` prefixes and since the `meta.id`, respectively `(task.ext.)prefix` is the same for the testdata, the alleleFrequency files get the exact same name `test.alleleFrequency_chr.txt` without the `tumour/normal` string and Ascat would exit with an error.

Reply:
It would only work if it was possible to assign two different meta.id's to the input files

Reply:
ok then all good. Maybe we can petition for some more flexible naming at some point :D